templates/taxi/car_list.html

```diff
@@ -1,22 +1,28 @@
 {% extends "base.html" %}
+{% load crispy_forms_filters %}
 
 {% block content %}
-  <h1>
-    Car list
-    <a href="{% url 'taxi:car-create' %}" class="btn btn-primary link-to-page">
-      Create
-    </a>
-  </h1>
-  {% if car_list %}
-    <ul>
-      {% for car in car_list %}
-        <li>
-          <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
-          {{ car.model }} ({{ car.manufacturer.name }})
-        </li>
-      {% endfor %}
-    </ul>
-  {% else %}
-    <p>There are no cars in taxi</p>
-  {% endif %}
+    <h1>
+        Car list
+        <a href="{% url 'taxi:car-create' %}" class="btn btn-primary link-to-page float-right">
+            Create
+        </a>
+    </h1>
+    <form method="get" action="" class="form-inline">
+        {{ search_form|crispy }}
+        <input type="submit" value="🔍" class="btn btn-secondary">
+    </form>
+    <br>
+    {% if car_list %}
+        <ul>
+            {% for car in car_list %}
+                <li>
+                    <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
```

Comment:
Ensure that the URL tag is correctly formatted. The space between `pk=car.id` and the closing quote might cause issues. It should be `{% url 'taxi:car-detail' pk=car.id %}` without any spaces.